examples/ir-ilu-preconditioned-solver/ir-ilu-preconditioned-solver.cpp

```diff
@@ -76,8 +76,16 @@ int main(int argc, char *argv[])
 
     // Read data
     auto A = gko::share(gko::read<mtx>(std::ifstream("data/A.mtx"), exec));
-    auto b = gko::read<vec>(std::ifstream("data/b.mtx"), exec);
-    auto x = gko::read<vec>(std::ifstream("data/x0.mtx"), exec);
+    // Create RHS and initial guess as 1
+    gko::size_type size = A->get_size()[0];
+    auto host_x = vec::create(exec->get_master(), gko::dim<2>(size, 1));
+    for (auto i = 0; i < size; i++) {
```

Comment:
```suggestion
    for (gko::size_type i = 0; i < size; i++) {
```
0 is `int` by default, so probably not 100% what you want here.